src/deepsparse/transformers/pipelines/token_classification.py

```diff
@@ -367,7 +367,7 @@ def route_input_to_bucket(
             input_schema.inputs,
             add_special_tokens=True,
             return_tensors="np",
-            padding=False,
+            padding=True,
```

Comment:
why are we adding padding here? if there's padding we cant extract the true sequence length, no?

Reply:
True. Let me further investigate the optimal solution to the problem, I have to admit that I was just quickly taking a look at the problem as a "side-project" and found out that more work is needed to understand it and to get the right solution.